packages/evolution-generator/src/tests/test_generate_questionnaire_dictionary.py

```
@@ -0,0 +1,136 @@
+]
+
+SECTIONS = {
+    "home": {
+        "title": "Home",
+        "abbreviation": "hm_",
+    }
+}
+
+
+class MockCell:
+    def __init__(self, value):
```

Ruff (0.16.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)